src/views/support.hbs

```diff
@@ -22,3 +22,27 @@
 {{#each agreements}}
 {{> terms/modal this}}
 {{/each}}
+
+<div class="container mb-3">
+  {{{__ "betterplace_text"}}}
+  <script type="text/javascript">
+    /* Configure at https://www.betterplace.org/de/projects/74137-unterstutze-die-lernwerkstatt-lernen-und-lernen-lassen-e-v/manage/iframe_donation_form/new */
+    var _bp_iframe = _bp_iframe || {};
+    _bp_iframe.project_id = 74137; /* REQUIRED */
+    _bp_iframe.lang = '{{{__ "locale"}}}'; /* Language of the form */
+    _bp_iframe.width = 600; /* Custom iframe-tag-width, integer */
+    _bp_iframe.color = 'fd7222'; /* Button and banderole color, hex without "#" */
+    _bp_iframe.background_color = 'ffffff'; /* Background-color, hex without "#" */
+    _bp_iframe.default_amount = 10; /* Donation-amount, integer 1-99 */
+    _bp_iframe.recurring_interval = 'monthly'; /* Interval for recurring donations, string out of ["single", "monthly", "quarter_yearly", "half_yearly", "yearly"] */
+    _bp_iframe.bottom_logo = true;
+    (function () {
+      var bp = document.createElement('script'); bp.type = 'text/javascript'; bp.async = true;
+      bp.src = 'https://betterplace-assets.betterplace.org/assets/load_donation_iframe.js';
+      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(bp, s);
+    })();
+  </script>
+  <div class="mt-3" id="betterplace_donation_iframe"
```

Comment:
Could we maybe make the page have two columns instead of a long scroll?

Reply:
No problem :)